db/const.go

```diff
@@ -81,6 +84,7 @@ var validRoles = map[UserRole]bool{
 	AdminRole:   true,
 	ManagerRole: true,
 	ViewerRole:  true,
+	AnyRole:     true,
```

Comment:
I think that `AnyRole` shouldn't be included in valid roles. If it is included a user can be included in a organization with this role. As far as i know, `AnyRole` should be only for internal use.